packages/icons/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@blueprintjs/icons",
+  "version": "1.0.0",
+  "description": "Components, fonts, icons, and css files for creating and displaying icons.",
+  "main": "dist/index.js",
+  "module": "dist/esm/index.js",
+  "typings": "dist/esm/index.d.ts",
+  "style": "dist/blueprint-icons.css",
+  "unpkg": "dist/icons.bundle.js",
+  "scripts": {
+    "compile": "npm-run-all -s 'compile:pre' -p 'compile:*'",
+    "compile:pre": "generate-icons-source",
+    "compile:esm": "tsc -p ./src",
+    "compile:cjs": "tsc -p ./src/tsconfig.cjs.json",
+    "compile:css": "sass-compile ./src",
+    "clean": "rm -rf dist/*",
+    "dev": "npm-run-all -p 'compile:esm -- --watch' 'compile:css -- --watch'",
+    "dist": "npm-run-all -s 'dist:*'",
+    "dist:bundle": "NODE_ENV=production webpack",
+    "dist:variables": "generate-css-variables generated/_icon-variables.scss",
+    "dist:verify": "assert-package-layout",
+    "lint": "npm-run-all -p lint:stylelint lint:tslint",
+    "lint:stylelint": "stylelint -s scss 'src/**/*.scss'",
+    "lint:tslint": "tslint '{src,test}/**/*.{ts,tsx}' -c ../tslint-config -t codeFrame",
+    "lint-fix": "yarn lint:tslint --fix",
+    "test": "npm-run-all -s compile:pre compile:cjs test:pre -p test:iso",
+    "test:pre": "tsc -p ./test",
+    "test:karma": "karma start",
+    "test:karma:debug": "karma start --single-run=false --reporters=mocha --debug",
```

Comment:
Let's remove these 2 scripts and remove `karma.conf.js` (remember to remove from `.npmignore`).